pkg/kubelet/kubelet_node_status.go

```diff
@@ -216,16 +218,47 @@ func (kl *Kubelet) reconcileCMADAnnotationWithExistingNode(node, existingNode *v
 	return true
 }
 
+// EnumerateDistribution resolves the underlying distribution
+func (kl *Kubelet) EnumerateDistribution() (string, string) {
+	file, err := os.Open("/usr/lib/os-release")
+	if err != nil {
+		return "", ""
+	}
+	defer file.Close()
+
+	id := ""
+	versionID := ""
+	scanner := bufio.NewScanner(file)
```

Comment:
Please write a test for this code. Maybe pass this file to another function?

```
func enumerateDistribution(r io.Reader) (id, versionID string)
```

Then write a test for that function so the test doesn't depend on an on disk file.